easybuild/easyconfigs/l/libzip/libzip-1.10.1-GCCcore-13.2.0.eb

```
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'libzip'
+version = '1.10.1'
+
+homepage = 'https://libzip.org/'
+description = "libzip is a C library for reading, creating, and modifying zip archives."
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/nih-at/libzip/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d56d857d1c3ad4a7f3a4c01a51c6a6e5530e35ab93503f62276e8ba2b306186a']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+]
+
+dependencies = [
+    ('OpenSSL', '3'),
```

Oops, forgot to specify the SYSTEM toolchain, and let's stick to 1.1 for now (other easyconfigs in this toolchain are using that version as well):
```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
```